tests/factories/PacbioSmrtLinkVersionFactory.js

```diff
@@ -0,0 +1,100 @@
+import { dataToObjectById } from './../../src/api/JsonApi'
+
+/*
+ * Factory for creating a list of smrt link versions
+ * @returns a base factory object with the smrt lin version data
+ * @returns a storeData object with the smrt link versions data stored by id
+ */
+const PacbioSmrtLinkVersionFactory = () => {
+  const data = {
+    data: [
+      {
+        id: '2',
+        type: 'smrt_link_versions',
+        links: {
+          self: 'http://traction/v1/pacbio/smrt_link_versions/2',
+        },
+        attributes: {
+          name: 'v11',
+          default: true,
+          active: true,
+        },
+        relationships: {
+          smrt_link_option_versions: {
+            links: {
+              self: 'http://traction/v1/pacbio/smrt_link_versions/2/relationships/smrt_link_option_versions',
+              related: 'http://traction/v1/pacbio/smrt_link_versions/2/smrt_link_option_versions',
+            },
+          },
+        },
+      },
+      {
+        id: '3',
+        type: 'smrt_link_versions',
+        links: {
+          self: 'http://traction/v1/pacbio/smrt_link_versions/3',
+        },
+        attributes: {
+          name: 'v12_revio',
+          default: false,
+          active: true,
+        },
+        relationships: {
+          smrt_link_option_versions: {
+            links: {
+              self: 'http://traction/v1/pacbio/smrt_link_versions/3/relationships/smrt_link_option_versions',
+              related: 'http://traction/v1/pacbio/smrt_link_versions/3/smrt_link_option_versions',
+            },
+          },
+        },
+      },
+      {
+        id: '4',
+        type: 'smrt_link_versions',
+        links: {
+          self: 'http://traction/v1/pacbio/smrt_link_versions/4',
+        },
+        attributes: {
+          name: 'v13_sequel_iie',
+          default: false,
+          active: true,
+        },
+        relationships: {
+          smrt_link_option_versions: {
+            links: {
+              self: 'http://traction/v1/pacbio/smrt_link_versions/4/relationships/smrt_link_option_versions',
+              related: 'http://traction/v1/pacbio/smrt_link_versions/4/smrt_link_option_versions',
+            },
+          },
+        },
+      },
+      {
+        id: '5',
+        type: 'smrt_link_versions',
+        links: {
+          self: 'http://traction/v1/pacbio/smrt_link_versions/5',
+        },
+        attributes: {
+          name: 'v13_revio',
+          default: false,
+          active: true,
+        },
+        relationships: {
+          smrt_link_option_versions: {
+            links: {
+              self: 'http://traction/v1/pacbio/smrt_link_versions/5/relationships/smrt_link_option_versions',
+              related: 'http://traction/v1/pacbio/smrt_link_versions/5/smrt_link_option_versions',
+            },
+          },
+        },
+      },
+    ],
+  }
+
+  const storeData = dataToObjectById({ ...data })
+
+  return { ...BaseFactory(data), storeData }
```

Comment:
I hope keeping store data will simplify unit tests a lot, as we replicate much of it across tests.

Reply:
Yes. I decide to add this at this level as each factory will create store data differently. We can see how it progresses ...

Reply:
Yes, it is best to keep it here because it varies a lot between stores and probably needs many steps for complicated stores. Maybe after we do all these, we might find some patterns and then we can do further abstraction.